docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -138,7 +139,8 @@ Step 1: Prepare The Install Environment
 
 #. Install ZFS in the Live CD environment::
 
-     apt install --yes debootstrap gdisk dkms dpkg-dev zfs-dkms
+     apt install --yes debootstrap gdisk dkms dpkg-dev zfs-dkms \
+         linux-headers-$(uname -r)
```

Comment:
No needs for it, because in live iso --install-recommends are activated and headers are coming from Recommends

Reply:
Thank for explanation.